docs/monoDrive_home/Simulator-Commands.md

```diff
@@ -0,0 +1,365 @@
+# Simulator API Commands
+
+## Get Simulator Version
+Retrieves the current version of the simulator and of the simulator API with the commands documented on this page.
+
+**command ID**:  "GetVersion"
```

Comment:
Should probably just make these subsections instead e.g. 
```
### Command ID
...
### Command Data
```

Reply:
The different sizing for the headings ended up looking a bit chaotic, but did capitalize these subheadings and bolded the entire command id line which has a similar effect.